Gemfile

```diff
@@ -38,7 +38,7 @@ gem 'execjs'
 gem 'twitter-bootstrap-rails', '~> 4.0'
 # Paginate
 gem 'will_paginate'
-gem 'will_paginate-bootstrap'
+gem 'bootstrap-will_paginate'
```

Comment:
__[RuboCop]__

Bundler/OrderedGems: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `bootstrap-will_paginate` should appear before `will_paginate`.


<!-- Comment by [SideCI](https://sideci.com) -->